docs/7.Scan Examples/Git History.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: default
+published: false
+title: SCA scanning
+nav_order: 20
+---
+
+# Git History
+Checkov supports scanning for secrets in git history, to identify and flag secrets which might not be in the current brunches, but are still visible by accessing the git history of the repo.
+Checkov fetches all the available commits, and uses the same strategy as in 'secrets' checkov framework to search for secrets in the diff available from git history.
+Each secret found, can be traced by the first commit in which it appeared, and the last commit, if it has been removed. 
+
+
+## Git History scanning
+
+Git history scan is the same as secrets scan.
+The only difference is that the root directory is the path to either the root git directory or the bare git repo.
+
```

Comment:
Need to add a line that say that `--scan-secrets-history` will only scan history for secrets and not other issues such as IaC misconfigs.

```suggestion
Using the `--scan-secrets-history` flag will scan git history for secrets only. This will not scan for other issues such as IaC misconfiguration. Use `--secrets-history-timeout` to set how long the secrets scan will run on history before stopping and returning results. The default is ___.
```

Reply:
@tsmithv11 
since we changed it to run multi process we can't really return the results.
we run in parallel and if it's not finished properly we won't be able to compose the results correctly

Reply:
Sounds good

```suggestion
Using the `--scan-secrets-history` flag will scan git history for secrets only. This will not scan for other issues such as IaC misconfiguration. Use `--secrets-history-timeout` to set how long the secrets scan will run on history before stopping and not returning results. The default is ___.
```